### ISSUE TITLE
Lifespan issues 2.0.0

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#1640]
1. The years should print in a scaled horizontal x-axes 
2. I look on my site for instance at the chart of I269 including his immediate family. I do not see all the relatives.
3. The selection box of Add individual is not clear. The horizontal line that we see is not clear.

Pre 2.0.0
![oldlifespans_2018-03-10_21-42-59](https://user-images.githubusercontent.com/5202996/37246149-7bd6087c-24ac-11e8-9c98-08f3d3396e26.jpg)

2.0.0
![2 0 0lifespans_2018-03-10_21-44-32](https://user-images.githubusercontent.com/5202996/37246150-84245eca-24ac-11e8-8269-7ca80426b662.jpg)

**Comment by fisharebest** [comment]
> The years should print in a scaled horizontal x-axes

They do for me.  Have you cleared your browser cache?

> the chart of I269 including his immediate family. I do not see all the relatives.

I see parents, spouses and children.  If you do not see all the relatives, what do you see instead (some, none, etc.?)

> The selection box of Add individual is not clear

This is the select2 plugin.  It needs styling.  It also needs some work, as the latest version contains a bug.  There's a separate issue for this.

**Comment by melizaa** [comment]
I cleaned now the cache and I see the x-axes. Thanks!

We used also to see siblings.

For me, I see my parents, my husband and my 2 sons, but not my daughter, who is the eldest child.  

For my father I86 I see parents, spouses and my 2 brothers. I do not see me.
I was born before my grandparents died.

For my grandfather I269 I see his parents, spouse and all the children.
The eldest child was born after her grandfather died. Her data is printed on the same line as his data.

For my great grandfather I271 I miss the eldest daughter Sarah.
In pre 2.0.0 her data is printed on the same row as the grandmother. The next son's data was printed on the same row as my great grandfather's sibling. He is now printed onn the row of the grandmother.

etc.
